### ISSUE TITLE
possible bug - confirm that singleton `proglang.txt` files containing only pyret do not produce duplicate courses

### ISSUE THREAD

**Issue by flannery-denny** [bootstrapworld/curriculum#1386]

**Comment by ds26gte** [comment]
If this is concerning the creation of both `algebra-2` and `algebra-pyret` courses, this has nothing to do with the build algorithm! There really are two separate pathways in the repo. As authors, you (@schanzer , @flannery-denny , @retabak ) have to take an executive decision about deleting one of these two.

**Comment by schanzer** [comment]
@ds26gte Flannery is not complaining about algebra-pyret

**Comment by flannery-denny** [comment]
True. On the `alg2` branch there was both an `algebra 2` course and an `algebra 2 pyret` course. I deleted the `proglang.txt` file per your directions @ds26gte. And you asked me to file a bug report because you didn't think having the `proglang.txt` file should have created 2 courses.
